### ISSUE TITLE
AWS plugin should support SSO

### ISSUE THREAD

**Issue by stuart-warren** [ohmyzsh/ohmyzsh#10113]
### If the feature request is for a plugin or theme, specify it here.

AWS plugin

### If the feature solves a problem you have, specify it here.

We use AWS SSO for login to AWS console currently the plugin does not support this

### Describe the proposed feature.

perform `aws sso login` if the profile has that configured

### Additional context

Currently the output is
```sh
❯ acp developer-48613280000
Obtaining session token for profile developer-48613280000

An error occurred (AccessDenied) when calling the GetSessionToken operation: Cannot call GetSessionToken with session credentials
```

**Comment by stuart-warren** [comment]
seems to be a duplicate of #10004

and similarly implemented in #9921 and #9795

**Comment by stuart-warren** [comment]
I'm probably just going to use my own shell alias instead
```sh
sap () {
        unset AWS_ACCESS_KEY_ID
        unset AWS_SECRET_ACCESS_KEY
        export AWS_PROFILE=$(aws configure list-profiles | fzf -0 )
        aws sts get-caller-identity > /dev/null 2>&1 || aws sso login
}
```
